src/pages/index.tsx

```diff
@@ -1,5 +1,101 @@
+import { Box, Button, Typography, useTheme } from '@mui/material';
+
+import { RegionCard } from '@/components';
+
+import {
+  AssignmentIcon,
+  InterlaceIcon,
+  PartitionIcon,
+  TransferIcon,
+} from '@/icons';
+
 const Home = () => {
-  return <></>;
+  const theme = useTheme();
+  const management = [
+    { label: 'partition', icon: PartitionIcon },
+    { label: 'interlace', icon: InterlaceIcon },
+    { label: 'transfer', icon: TransferIcon },
+    { label: 'assign', icon: AssignmentIcon },
+  ];
+
+  return (
+    <Box sx={{ display: 'flex', height: '100%', gap: '1rem' }}>
+      <Box sx={{ maxWidth: '45rem', flexGrow: 1, overflow: 'auto' }}>
+        <Box>
+          <Typography
+            variant='subtitle2'
+            sx={{ color: theme.palette.text.secondary }}
+          >
+            Manage your cores
+          </Typography>
+          <Typography
+            variant='subtitle1'
+            sx={{ color: theme.palette.text.primary }}
+          >
+            Regions Dashboard
+          </Typography>
+        </Box>
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            gap: '2rem',
+            mt: '1rem',
+          }}
+        >
+          {[1, 2].map((_, index) => (
+            <RegionCard
+              key={index}
+              region={{
```

Comment:
After getting [this](https://github.com/RegionX-Labs/Coretime-Mock/pull/19) merged we can have actual data here. Do you want to make that change in this PR, or do that separately?

Reply:
we can remove this block for merging the PR.
it's just for demonstration purpose only.